<?xml version="1.0"?>
<launch>

  <rosparam command="load" file="$(find outdoor_waypoint_nav)/params/ekf_params.yaml" />
  <rosparam command="load" file="$(find outdoor_waypoint_nav)/params/navsat_params_sim.yaml" />
  
  <node pkg="robot_localization" type="ekf_localization_node" name="ekf_se_odom" clear_params="true"/>

  <node pkg="robot_localization" type="ekf_localization_node" name="ekf_se_map" clear_params="true">
    <remap from="odometry/filtered" to="odometry/filtered_map"/>
  </node>

  <node pkg="robot_localization" type="navsat_transform_node" name="navsat_transform" clear_params="true" output="screen">
      <remap from="odometry/filtered" to="odometry/filtered_map"/>
      <remap from="gps/fix" to="navsat/fix"/>
  </node>

</launch>
